src/modules/accessory.json

```diff
@@ -0,0 +1,202 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "Accessory",
+    "description": "A module for assisting pairing and managing Firebolt compatible accessories.",
+    "version": "0.0.0"
+  },
+  "components": {
+    "schema": {
+      "AccessoryList": {
+        "title": "AccessoryList",
+        "type": "object",
+        "description": "Contains a list of Accessories paired to the device.",
+        "properties": {
+          "list": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schema/AccessoryInfo"
+            }
+          }
+        }
+      },
+      "AccessoryInfo": {
+        "title": "AccessoryInfo",
+        "description": "Properties of a paired accessory.",
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "description": "Type of accessory",
+            "enum": ["remote", "speaker"]
+          },
+          "make": {
+            "type": "string",
+            "description": "Name of the manufacturer of the accessory"
+          },
+          "model": {
+            "type": "string",
+            "description": "Model name of the accessory"
+          },
+          "connectivity": {
+            "type": "string",
+            "description": "Mechanism through which the accessory is connected to the device. Eg Bluetooth, ZRC, RF4Ce"
+          }
+        }
+      }
+    }
+  },
+  "methods": [
+    {
+      "name": "pair",
+      "summary": "Pair an accessory with the device.",
+      "params": [
+        {
+          "name": "type",
+          "summary": "Defines the type of device being paired",
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+            "name": "connectivity",
+            "summary": "Defines the mechanism by which accessory will connect to the device.",
+            "schema": {
+              "type": "string"
+            }
+        },
+        {
+          "name": "timeoutInSecs",
```

Comment:
I like how the name specifies that the unit is seconds... however, we haven't done that in other Firebolt parameter/field names.

I suggest calling it `timeout` for now, and writing a proposal on standardizing *all* of our timeout property names.

Reply:
changed it to timeout